index.mdx

```diff
@@ -56,8 +56,8 @@ To help you find content in the most effective way, Scaleway has organized its c
 
 The goal of this document is to provide clients with information about every resource in the Scaleway ecosystem and the procedures they should follow to get the best experience from the Scaleway services.
 
-[Scaleway Quickstart Guide](scaleway-quickstart-3.0.pdf)
-: Learn more about Scaleway products through the **Products Documentation PDF**. 
+[Scaleway Quickstart Guide](scaleway-quickstart-3.2.pdf)
+: Learn more about Scaleway products through the **Products Documentation PDF**.
```

Comment:
I feel like this should probably be "Product (singular) Documentation PDF" - what do you think?

Reply:
```suggestion
: Learn more about Scaleway products through this PDF guide.
```